src/app/util/af-types.h

```diff
@@ -245,6 +245,12 @@ struct EmberAfDefinedEndpoint
 #endif
 };
 
+enum class MatterClusterShutdownType
```

Comment:
Quick question: can we not just include ServerClusterInterface.h here and use the same enum for both?    Or put the enum in some header that we can include from both places?

Having the two separate enums seems a bit annoying.

Followup is fine for merging them, though; don't need to block on it.